eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -70,12 +70,15 @@ function GetAdjustedReadmeContent($pkgInfo, $lang){
     }
 
     $fileContent = $pkgInfo.ReadmeContent
-    $foundTitle = ""
 
     # only replace the version if the formatted header can be found
-    $titleRegex = "(\# Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C))"
-    $fileContent = $pkgInfo.ReadmeContent -replace $titleRegex, "`${1} - Version $($pkgInfo.PackageVersion) `n"
-    
+    $titleRegex = "(\#\s+(?<filetitle>Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C)))"
```

Comment:
Lets please be sure to do some testing of the regex and script logic locally with a few readme examples.

Reply:
Already passed the test in local. I am setting up the template pipeline for test.

Reply:
Verified through template:
https://github.com/Azure/azure-docs-sdk-java/pull/1255/files